roles/openshift_ovirt/tasks/build_vm_list.yml

```diff
@@ -46,7 +52,13 @@
       'dns_search': '{{ item["dns_search"] }}',
         {% endif -%}
       {% elif item.count > 1 -%}
+      # This allows you to use the DHCP-DNS given names with a different VM name
+        {% if item.empty_hostname is defined and item.empty_hostname -%}
+      'host_name': '',
+        {% else -%}
       'host_name': '{{ item.name }}{{ iter }}.{{ openshift_ovirt_dns_zone }}',
```

Comment:
Same, this would best be handled by `ternary`

Reply:
:+1: